src/components/inputs/LInputLoaded.vue

```diff
@@ -142,6 +142,11 @@ export default {
       const { large, small, $attrs } = this
 
       return getInputHeight({large, small, custom: $attrs.height})
+    },
+    cssVars() {
```

Comment:
```suggestion
    cssVars () {
```
Seria bom pensarmos em outro nome pra essa computed, algo como varsInputloaded ou algo nesse sentido talvez seja melhor

Reply:
mas essa computed já ta no inputLoaded, não seria necessário especificar